client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
@gwwar mostly, yeah. unfortunately I think I have a case where a pure URL dedupe isn't necessarily enough.

When we ask for updates to a stream, we ask for a after our most recent post and after the client's Date.now(). Since we use the client's `now`, the URL changes every time our update poller triggers. I think we're mostly safe from dedupes here for other reasons, but something to consider.